src/sandbox/mod.rs

```diff
@@ -60,6 +60,11 @@ pub trait Pod {
     fn remove(&mut self, _: &SandboxData) -> Result<()> {
         Ok(())
     }
+
+    // Returns whether a sandbox is ready or not
```

Comment:
nit: I guess we will block start and remove if the sandbox is not ready, right? We may add this as doc comment, too.